docs/3-Deploy-to-Kool-Cloud/2-kool.deploy.yml-Reference.md

```diff
@@ -0,0 +1,139 @@
+          chown: user:group
+
+    # By default, Kool Cloud will rollout new deployments in a blue-green fashion.
+    # If you want to disable it and make sure the current running container
+    # stopped before the new one is created, set 'recreate: true'.
+    recreate: false
+
+    # Sometimes you may make changes to your app that wouldn't necessarily trigger
+    # new containers to be created by the kool cloud deploy process.
+    #
+    # For example, if you change only environment variables or use a fixed image tag
+    # like ':latest' that doesn't change.
+    #
+    # By setting `force: true`, you tell the API to always update this service.
+    force: false
+
+    # Here we can define processes that behave like services and must be run in the cloud only.
+    # Usually, this is very helpful for queue workers and that kind of stuff. The processes will
+    # run as other containers in your deployment using the very same image as the main service.
+    daemons:
+      - command: [ start-queues, arg1, arg2 ]
+      # As mentioned above in the 'public' config, you may also want to have a daemon
+      # that serves specific requests. A common use case is WebSocket services. You can
+      # define that your daemon exposes a TCP port with the 'expose' entry.
+      - command: [ run-websocker-server, --port=3000 ]
+        expose: 3000
+
+    # Hooks
+    #
+    # It's possible that you want to run some extra steps either before or after every
+    # time your application is deployed. Such hooks are executed in standalone containers
+    # using the same Docker image about to be deployed.
+
+    #
+    # The 'before' hook is a special section where we can define commands to be executed
+    # right before a new deployment happens.
+    before:
+      - script_to_run.sh
+    # The 'after' hook is a special section where we can define procedures to be executed
+    # right after a new deployment finishes.
+    after:
+      - [ run-database-migrations, arg1, arg2 ]
+
+    # The 'cron' config allows you to set scheduled actions to run in a Cron job fashion.
+    cron:
+      - { schedule: "* * * * *", command: [ run-some-task-every-minute ] }
+
+    # Environment Variables to the cloud container
+    #
+    # You usually will want to provide a file containing the environment variables
+    # that your deployment should have. Such a file may contain special Kool variables
+    # that will be translated to their actual value by the Deploy API.
+    #
+    # There are two ways to provide such a file.
+    #
+    # 'environment' refers to a file with all of the environment variables available.
+    # The Deploy API will get the contents of this file and make all of them available
+    # as true environment variables for the container. You can check them in your deployed
+    # container via kool cloud exec env (assuming you have env available).
+    environment: kool.deploy.env
```

Comment:
i think we need to fix this before launch `environment` and `env`...  it's very confusing...